### PR TITLE
CORE-4876 Edit security policy to sensible DB Sandbox policy

### PR DESCRIPTION
**Pull request by mbrkic-r3**
-  Added permissions for Persistence Sandbox to Security Manager policy file

### CODE REVIEW

File: components/security-manager/src/main/resources/basic_security.policy [hunk 1]
```diff
@@ -717,7 +721,806 @@ DENY {
 #
 (javax.xml.ws.WebServicePermission "publishEndpoint" "")
 
-} "Basic security profile"
+} "Basic security profile for FLOW Sandbox"
+
+#
+# Permissions for PERSISTENCE Sandbox
+#
+
```
Comment (chrisr3) [inline components/security-manager/src/main/resources/basic_security.policy:729]: Are there any specific `net.corda.v5.*` services or packages that we wish to DENY for `PERSISTENCE/*` sandboxes? E.g.
```
DENY {
[org.osgi.service.condpermadmin.BundleLocationCondition "PERSISTENCE/*"]

(org.osgi.framework.PackagePermission "net.corda.v5.application.crypto", "import")
(org.osgi.framework.ServicePermission "net.corda.v5.application.flows.FlowEngine", "get")
} "Databases don't do some things"
```

**Review by chrisr3 — COMMENTED** [review]
These files are all so big that I cannot possibly tell if anything is missing or mis-copied :cry:.

File: components/security-manager/src/main/resources/basic_security.policy (hunk 1 above)
Comment (mbrkic-r3) [reply components/security-manager/src/main/resources/basic_security.policy:729]: It's a matter of knowing all services that are available and which sandbox uses which services. Do we know this already, can we compile a table that specifies that and can be then transformed to permissions?

**Comment by mbrkic-r3** [comment]
> These files are all so big that I cannot possibly tell if anything is missing or mis-copied cry.

Yes, unfortunately it's a tricky issue, we could split permissions into multiple file but I'm afraid it might still be hard to tell if something is missing. However, then we also introduce the risk of ordering them incorrectly (order of permissions is very important).

**Review by driessamyn — APPROVED** [review]
LGTM.

We should review this again post DP2.

**Pull request merged**